### ISSUE TITLE
I can't build QGIS3 due Qt5WebKit version. It's demanding 5.11.x and I have 5.12

### ISSUE THREAD

**Issue by luispuerto** [OSGeo/homebrew-osgeo4mac#546]
This is the error: 

```
cmake -G Ninja -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/usr/local
Last 15 lines from /Users/lpuerto/Library/Logs/Homebrew/qgis3/30.cmake:
  Found package configuration file:

    /usr/local/opt/qt5-webkit/lib/cmake/Qt5WebKit/Qt5WebKitConfig.cmake

  but it set Qt5WebKit_FOUND to FALSE so package "Qt5WebKit" is considered to
  be NOT FOUND.  Reason given by package:

  Qt5WebKit could not be found because dependency is required to have exact
  version 5.11.x.
```

I updated today: 

```
==> Upgrading 6 outdated packages:
postgresql 10.6 -> 11.1, nettle 3.4 -> 3.4.1, qt 5.11.2_1 -> 5.12.0, numpy 1.15.4 -> 1.15.4_1, openblas 0.3.3 -> 0.3.4, postgis 2.5.0 -> 2.5.0_1
==> Upgrading qt
==> Downloading https://homebrew.bintray.com/bottles/qt-5.12.0.high_sierra.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/b5/b59a8686dcb7b95e14ec4c78a8aa989e8d8cbb5837618f3afbd5d42566c98c2b?__gd
######################################################################## 100.0%
==> Pouring qt-5.12.0.high_sierra.bottle.tar.gz
==> Caveats
We agreed to the Qt open source license for you.
If this is unacceptable you should uninstall.

qt is keg-only, which means it was not symlinked into /usr/local,
because Qt 5 has CMake issues when linked.

If you need to have qt first in your PATH run:
  echo 'export PATH="/usr/local/opt/qt/bin:$PATH"' >> ~/.zshrc

For compilers to find qt you may need to set:
  export LDFLAGS="-L/usr/local/opt/qt/lib"
  export CPPFLAGS="-I/usr/local/opt/qt/include"

For pkg-config to find qt you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/qt/lib/pkgconfig"

==> Summary
🍺  /usr/local/Cellar/qt/5.12.0: 9,689 files, 318.9MB
```

**Comment by luispuerto** [comment]
PS/ I rolled back to my previous version of QT with `brew switch qt 5.11.2_1` and now it's working my previous install of QGIS3. But seems that something is wrong either in our formula or in the cmake instructions.

**Comment by fjperini** [comment]
@luispuerto It is necessary to rebuild `qt5-webkit` for `qt 5.12`.

Before I will apply a small change to that formula.

**Comment by fjperini** [comment]
@luispuerto Changes applied!

**Comment by luispuerto** [comment]
Great!! if they are merged before tonight I'll try to test them... (it's really a pain the 😶 to build....)

**Comment by luispuerto** [comment]
@fjperini are you serious? @nickrobison are you building them your computer and then uploading them to bintray? 

For the building time is why I was suggesting the other day (#541) that would be wonderful to have bottles with options, so we can get all the rings and whistles without need to build. 

I like to have the last version of the soft, but I wouldn't mind to wait a couple of hours if the CI can get for me a ready to pour bottle. 

With R, we have the same problem, and even we need to have a separate [tap](https://github.com/sethrfore/homebrew-r-srf) with an R with all the options, so we can build and R with OpenBLAS and all the support. However, R is a much lighter software.

**Comment by fjperini** [comment]
@luispuerto if it is not built in Travis CI, it must be done manually.

Of course, it would be good to have those possibilities when using Circle CI.

**Comment by nickrobison** [comment]
@luispuerto @fjperini I believe I resolved this with 8bf92dd. Let me know if you're still having issues. Sorry for the delay in getting this fixed. I verified that webkit builds successfully in CircleCI, so once those changes are merged in, things should get much easier,.
